youtube_dl/extractor/vimpothr.py

```diff
@@ -0,0 +1,62 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+
+class VimpOTHRVideoIE(InfoExtractor):
+    _VALID_URL = r'https?://vimp\.oth-regensburg\.de/video/(?P<title>[a-zA-Z0-9_-]+)/(?P<id>[a-z0-9]{32})'
+    _TEST = {
+        'url': 'https://vimp.oth-regensburg.de/video/VE2-Vorlesungsaufzeichnung-von-Fr-5November/ca47954b51baa0f0fb1481cc4df0558a',
+        'md5': 'eabb43c7bcc884773b0b0d2e37ebae87',
+        'info_dict': {
+            'id': 'ca47954b51baa0f0fb1481cc4df0558a',
+            'ext': 'mp4',
+            'title': 'VE2-Vorlesungsaufzeichnung von Fr. 5.November',
+            'description': 'Invarianzsatz, Rentenbarwerte',
+            'thumbnail': 'https://vimp.oth-regensburg.de/cache/b56c79d598c594b117d769dc1731ed6a.jpg',
+        }
+    }
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
+        title = self._html_search_regex(r'<h1>(.+?)</h1>', webpage, 'title') or self._og_search_title(webpage)
```

Comment:
`_html_search_regex()` throws unless given a `default`(returns it silently) or `fatal=False` (returns `None` with a warning), so the OG search won't execute.

Reply:
I'm wondering if it would make sense to rely only on _og_search. What would you recommend?

Reply:
It depends what content you typically find in the first <h1> element vs the content attribute of the <meta thingy=og:title> element.

But maybe you could actually ask someone at OTH what their policy on metadata is. If you got a useful answer, you could treat what they say as the first target and add fall-backs for any of your test videos that don't comply.